spec/helpers/account/tasks_helper_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Account::TasksHelper, type: :helper do
+  let!(:user)         { create(:user) }
+  let!(:workspace)    { create(:workspace, user: user) }
+  let!(:project)      { create(:project, workspace: workspace, users: [user]) }
+
+  context 'section' do
+    let!(:task) { create(:task, project: project, section: true) }
+
+    subject { helper.task_link_class(task) }
+  
+    it { is_expected.to include('title') }
+    it { is_expected.to include('is-4') }
+    it { is_expected.to include('is-italic') }

```

Comment:
nitpick `its(:split) { is_expected.to contain_exactly('title', 'is-4', 'is-italic') }`